typescript/src/accounts/evm/resolveViemClients.ts

```diff
@@ -164,15 +203,20 @@ export async function resolveViemClients(
 
   // If it's a valid network identifier, use the mapping
   if (isNetworkIdentifier(networkOrNodeUrl)) {
+    const rpcUrl =
+      networkOrNodeUrl === "base" || networkOrNodeUrl === "base-sepolia"
+        ? await getBaseNodeRpcUrl(networkOrNodeUrl)
+        : undefined;
+
     chain = resolveNetworkToChain(networkOrNodeUrl);
     const publicClient = createPublicClient({
       chain,
-      transport: http(),
+      transport: http(rpcUrl),
```

Comment:
so we'll intentionally use the default viem nodes for the chain if the rpcUrl is undefined?

Reply:
yep that's correct